gno.land/cmd/gnoland/start.go

```diff
@@ -41,32 +37,23 @@ type gnolandCfg struct {
 	config                string
 }
 
-func main() {
-	cfg := &gnolandCfg{}
+func newStartCmd(io *commands.IO) *commands.Command {
+	cfg := &startCfg{}
 
-	cmd := commands.NewCommand(
+	return commands.NewCommand(
 		commands.Metadata{
-			ShortUsage: "[flags] [<arg>...]",
-			LongHelp:   "Starts the gnoland blockchain node",
-			Options: []ff.Option{
-				ff.WithConfigFileFlag("config"),
-				ff.WithConfigFileParser(fftoml.Parser),
-			},
+			Name:       "start",
+			ShortUsage: "start [flags]",
+			ShortHelp:  "run the full node",
```

Comment:
Capitalize the first letter?
```suggestion
			ShortHelp:  "Run the full node",
```